1/debian9/1.16/Dockerfile

```diff
@@ -0,0 +1,45 @@
+# WARNING: This Dockerfile installs NGINX Prometheus Exporter and should be used
+# only in versions 1.14.x and 1.15.x. For newest version of NGINX, use template
+# that doesn't install exporter and use exporter as sidecar.
+# For more information, see exporter directory.
+
+FROM gcr.io/google-appengine/debian9
+
+ENV NGINX_VERSION 1.16.1*
+ENV C2D_RELEASE 1.16.1
+
+RUN set -x \
+        && apt-get update \
+        && apt-get install -y \
+                dirmngr \
+                gnupg \
+                wget \
+        && rm -rf /var/lib/apt/lists/*
+
+RUN set -x \
+        && echo "deb http://nginx.org/packages/debian/ stretch nginx" >> /etc/apt/sources.list \
+        && wget http://nginx.org/packages/keys/nginx_signing.key \
+        && apt-key add nginx_signing.key \
+        && apt-get update \
+        && apt-get install --no-install-recommends --no-install-suggests -y \
+                                                ca-certificates \
+                                                nginx=${NGINX_VERSION} \
+                                                nginx-module-xslt=${NGINX_VERSION} \
+                                                nginx-module-geoip=${NGINX_VERSION} \
+                                                nginx-module-image-filter=${NGINX_VERSION} \
+                                                nginx-module-perl=${NGINX_VERSION} \
+                                                nginx-module-njs=${NGINX_VERSION} \
+                                                gettext-base \
+        && rm -rf /var/lib/apt/lists/*
+
+# forward request and error logs to docker log collector
+RUN ln -sf /dev/stdout /var/log/nginx/access.log \
+        && ln -sf /dev/stderr /var/log/nginx/error.log
+
+COPY stub.conf /etc/nginx/conf.d/stub.conf
```

Comment:
This config is specific to NGINX Prometheus Exporter. Could we not include this into non-exporter image?

Reply:
Done.